pgc_ortho.py

```diff
@@ -302,6 +309,7 @@ def main():
             for k, v in results.items():
                 if v != 0:
                     logger.warning("Failed Image: %s", k)
+                    sys.exit(1)
```

Comment:
Adding a sys.exit here will result in the program aborting before it has reported all the failed images.  Note that the code can run on a set of images at once.  The code should find the error here, but not exit with a non-zero return until after the last line of the existing code.

```
        for k, v in results.items():
              if v != 0:
                  logger.warning("Failed Image: %s", k)
                  ret_code = 1

logger.info("Done")
sys.exit(ret_code)
```

You'll also need to add `ret_code = 0` around line 287 incase no errors were found..